index.js

```diff
@@ -14,13 +22,31 @@ if(process.env.NODE_ENV!="production"){
 
 const app = express();
 
+app.use(express.json());
+app.use(express.urlencoded({extended:false}));
+
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
 
 app.get('/', (req, res) => res.render('pages/index'))
 
+app.get('/restaurant/profile/:uid', (req, res) => {
+  var uid = req.params.uid;
+  console.log(uid);
+  var query = `select * from restaurants where id=${uid}`;
```

Comment:
I am getting a database connection error, the parameters should be labelled $1 and passed as second arg in the query

Reply:
UPDATE: This syntax now works by placing require('dotenv') before connection string definition